ansible/roles/products/teleop/server/desktop-icons/tasks/real-robots.yml

```diff
@@ -259,6 +298,90 @@
     desktop_icon_path: "Shadow Advanced Launchers/4 - Launch Bimanual Teleop GUI"
   when: bimanual|bool
 
+- name: Install desktop icon for remote right side teleop GUI
+  import_tasks: roslaunch-icon.yml
+  vars:
+    desktop_icon_png: "{{ teleop_glove_right }}"
+    launch_script: "shadow_GUI_right_remote.sh"
+    project_name_input: "{{ project_name_input_value }}"
+    launch_file_input: "{{ launch_file_input_gui_right_realhw }} remote:=true"
+    desktop_icon_name: "4 - Launch Remote Right Teleop GUI"
+    desktop_icon_path: "Shadow Advanced Launchers/4 - Launch Remote Right Teleop GUI"
+  when:
+    - bimanual|bool or hand_side=="right"
+    - remote_icons|bool
+    - glove=="shadow_glove"
+
+- name: Install desktop icon for remote left side teleop GUI
+  import_tasks: roslaunch-icon.yml
+  vars:
+    desktop_icon_png: "{{ teleop_glove_left }}"
+    launch_script: "shadow_GUI_left_remote.sh"
+    project_name_input: "{{ project_name_input_value }}"
+    launch_file_input: "{{ launch_file_input_gui_left_realhw }} remote:=true"
+    desktop_icon_name: "4 - Launch Remote Left Teleop GUI"
+    desktop_icon_path: "Shadow Advanced Launchers/4 - Launch Remote Left Teleop GUI"
+  when:
+    - bimanual|bool or hand_side=="left"
+    - remote_icons|bool
+    - glove=="shadow_glove"
+
+- name: Install desktop icon for remote bimanual teleop GUI
+  import_tasks: roslaunch-icon.yml
+  vars:
+    desktop_icon_png: "{{ teleop_glove_bimanual }}"
+    launch_script: "shadow_GUI_bimanual_remote.sh"
+    project_name_input: "{{ project_name_input_value }}"
+    launch_file_input: "{{ launch_file_input_gui_bimanual_realhw }} remote:=true"
+    desktop_icon_name: "4 - Launch Remote Bimanual Teleop GUI"
+    desktop_icon_path: "Shadow Advanced Launchers/4 - Launch Remote Bimanual Teleop GUI"
+  when:
+    - bimanual|bool
+    - remote_icons|bool
+    - glove=="shadow_glove"
+    
+- name: Install desktop icon for remote right side teleop GUI
```

Comment:
this is exactly the same as above no matter if its haptx or shadow glove, isnt it?

Reply:
For shadow glove we have the **remote** flag controlling the launch of pedal and vive locally, for haptx there's no 'remote' flag so I am setting the pedal not to be launched locally with haptx.

Reply:
ok, approved